VelocitySDK/Sensors/VLTSensorBuilder.m

```diff
@@ -37,7 +39,17 @@ + (BOOL)isSimulator
     gyroRecorder = [[VLTSimulatorSensorRecorder alloc] initWithUpdateInterval:updateInterval
                                                                  timeInBuffer:bufferSize
                                                                    sensorType:VLTSensorTypeGyro];
-    return [[VLTMultiMotionRecorder alloc] initWithMotionRecorders:@[accRecorder, gyroRecorder]];
+    NSMutableArray<VLTSensorRecorder *> *recorders = [[NSMutableArray alloc] init];
+    [recorders addObject:accRecorder];
+    [recorders addObject:gyroRecorder];
+
+    if ([VLTGPS isEnabled]) {
+        VLTGPSSensorRecorder *gpsRecorder = [[VLTGPSSensorRecorder alloc] initWithUpdateInterval:updateInterval
+                                                                                    timeInBuffer:bufferSize];
+        [recorders addObject:gpsRecorder];
+    }
```

Comment:
Right now this case is not handled, but I'm not sure if we even should handle it.